modules/story_factions/README.md

```diff
@@ -1,5 +1,25 @@
+			  Story Factions by Jason Hood
+
+				  Version 1.00
+
+
+Story Factions allows customization of the factions that are excluded from the
+reputation list.  Using this DLL defines all such factions, completely replacing
+the defaults (provided the ini exists and actually contains factions).	To use
+it, copy STORYFACTIONS.DLL to the EXE directory and add it to the [Libraries]
+section of EXE\dacom.ini.  Then copy STORYFACTIONS.INI to the DATA directory and
+edit it as required.
+
```

Comment:
I suggest removing this section and simply adding `Original version written by Jason Hood, 23 March, 2010`.
This currently serves as instructions which are incorrect.